src/cljs/witan/ui/controllers/workspace.cljs

```diff
@@ -39,12 +39,12 @@
        [:function/name
         :function/id
         :function/version]}
-    (data/query `[{[:workspace/list-by-owner ~id]
-                   [:workspace/name
-                    :workspace/id
-                    :workspace/owner-name
-                    :workspace/modified]}]
-                on-receive)))
+    #_(data/query `[{[:workspace/list-by-owner ~id]
```

Comment:
All these queries are broken (the whole of workspaces is broken) and the devcards build was firing some of them, causing errors. They all need to be entirely changed.